src/presence_analyzer/static/js/mean_time_weekday.js

```diff
@@ -0,0 +1,49 @@
+google.load("visualization", "1", {packages:["corechart"], 'language': 'pl'});
+
+function parseInterval(value) {
```

Comment:
DRY rule - that's now placed in each screen as far as i remember, extract it to separate file.